packages/@ourworldindata/explorer/src/ExplorerProgram.ts

```diff
@@ -547,19 +661,23 @@ const parseColumnDefs = (block: string[][]): OwidColumnDef[] => {
             (row) => !!(row.slug || typeof row.owidVariableId === "number"),
             "Keep only column defs with a slug or variable id"
         )
+
     return columnsTable.rows.map((row) => {
-        // ignore slug if a variable id is given
+        // Ignore slug if a variable id is given
         const hasValidVariableId =
             row.owidVariableId && isNotErrorValue(row.owidVariableId)
         if (hasValidVariableId && row.slug) delete row.slug
 
+        // Add display properties to the 'display' object
         for (const field in row) {
             const cellDef = ColumnGrammar[field]
-            if (cellDef?.display) {
-                // move field into 'display' object
+            if (
+                cellDef?.isDisplayProperty &&
+                isNotErrorValueOrEmptyCell(row[field]) &&
+                row[field] !== ""
+            ) {
                 row.display = row.display || {}
                 row.display[field] = row[field]
-                delete row[field]
```

Comment:
Not deleting the field from the top level feels a little safer to me, but it might be ok to delete it as well. Not sure. That's something I'd want to look into after my holiday